db/migrate/20160808150745_add_metrics_status_to_ext_management_system.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsStatusToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_metrics_error, :text
```

Comment:
This name is bad since it can contain 'Invalid', 'Unavailable' or 'Valid' which is not an error. I suggest: `metrics_state`  - the word state implies 'last'.